apps/passport-server/src/database/queries/telegram/insertTelegramConversation.ts

```diff
@@ -7,15 +7,15 @@ import { sqlQuery } from "../../sqlQuery";
 export async function insertTelegramVerification(
   client: Pool,
   telegramUserId: number,
-  telegramChatId: number
+  telegramChatId: number,
+  semaphoreId: string
 ): Promise<number> {
   const result = await sqlQuery(
     client,
     `\
-insert into telegram_bot_conversations (telegram_user_id, telegram_chat_id, verified)
-values ($1, $2, true)
-on conflict do nothing;`,
```

Comment:
Why remove this?

Reply:
We have two unique index constraints that govern this table: 1. That there cannot be a duplicate (user_id, chat_id) entry. (A user cannot join a chat twice). 2. There cannot be a duplicate (semaphore_id, chat_id) mapping. (A user cannot re-use a semaphore id with a different account). If we have `do nothing` on conflict, no error will be thrown when a user violates one of the above constraints. I believe that `doing nothing` is still ok behavior because it just means an invalid entry won't be added, but I prefer throwing an error and informing the user what has happened.